wasm.md

```diff
@@ -764,12 +762,11 @@ The `#take` function will return the parameter stack in the reversed order, then
 ```k
     syntax PlainInstr ::= "call" Index
  // ----------------------------------
-    rule <k> call TFIDX => ( invoke FADDR:Int ) ... </k>
+    rule <k> call IDX:Int => ( invoke FADDR:Int ) ... </k>
          <curModIdx> CUR </curModIdx>
          <moduleInst>
            <modIdx> CUR </modIdx>
-           <funcIds> IDS </funcIds>
-           <funcAddrs> ... #ContextLookup(IDS , TFIDX) |-> FADDR ... </funcAddrs>
+           <funcAddrs> ... IDX |-> FADDR ... </funcAddrs>
```

Comment:
I think it makes more sense to put the `FADDR:Int` in the `<funcAddrs>` cell, not the `<k>` cell, because it shows up on the LHS in the `<funcAddrs>` cell and the rHS in the `<k>` cell.

Reply:
(the sort check, that is)

But also, by putting it in an `(invoke FADDR)`, you're asserting that it's the same sort that `(invoke_)` takes, which is either `Index` or `Int` (not sure), so that alone may be enough.

Reply:
`invoke` takes `Int` so the sort check is unnecessary